backend/dataall/modules/dataset_sharing/services/share_object_service.py

```diff
@@ -236,7 +247,7 @@ def approve_share_object(cls, uri: str):
             session.add(approve_share_task)
 
         Worker.queue(engine=context.db_engine, task_ids=[approve_share_task.taskUri])
-
+        print('Hurray!')
```

Comment:
Great print statement :) ... but to remove haha

Reply:
Oops. Removed)